lib/liquid/tags/assign.rb

```diff
@@ -45,6 +47,12 @@ def assign_score_of(val)
         1
       end
     end
+
+    class ParseTreeVisitor < Liquid::ParseTreeVisitor
+      def children
+        [@node.from]
```

Comment:
why is `@node.to` not also considered a child?

Reply:
`@node.to` is just the name of the variable being assigned, it's not an expression